python/tutorials/06-fused-attention.py

```diff
@@ -111,6 +112,11 @@ def _host_descriptor_pre_hook(nargs):
     for s in NUM_STAGES_OPTIONS \
     for w in [4, 8]\
 ]
+if "pytest" in sys.modules:
+    # Use a single config in testing for reproducibility
+    configs = [
+        triton.Config(dict(BLOCK_M=64, BLOCK_N=64), num_stages=4, num_warps=4, pre_hook=_host_descriptor_pre_hook),
```

Comment:
so now the functional test will test something different than what we use for performance. This isn't great :/ 

Would be nice to test all the config instead but I'm not sure if there is an easy way to do that